src/main/java/core/basesyntax/service/handlerfruits/PurchaseFruitsHandlerImp.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service.handlerfruits;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class PurchaseFruitsHandlerImp implements FruitsHandler {
+    @Override
+    public void handle(FruitTransaction fruitTransaction) {
+        Integer lastSum = Storage.fruits.get(fruitTransaction.getFruitName());
```

Comment:
lets throw an exception when there is not enough fruits

Reply:
not fixed